.github/workflows/build.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: |
           go install github.com/jstemmer/go-junit-report@v1.0.0
           export CI="true"
-          sudo -E make CLOUD_PROVIDER=${{ matrix.provider }} test | tee tests_report.txt
+          sudo -E env PATH="$PATH" make CLOUD_PROVIDER=${{ matrix.provider }} test | tee tests_report.txt
```

Comment:
Sorry for my ignorance, but is `sudo -E env PATH="$PATH"` doing anything that `sudo -E PATH="$PATH"` wouldn't? That the pattern I see a lot more commonly in our code

Reply:
My understanding was also that `sudo -E PATH="$PATH"` should do what we want. Even `-E` should have been sufficient since it's supposed to preserve environment. But looks like it doesn't. See the examples - https://github.com/confidential-containers/cloud-api-adaptor/issues/165
It might be to do with version of sudo, shell etc. Honestly I don't know :-(

Reply:
Ok little more digging and google helped.

The behaviour is due to the `env_reset` and `secure_path` options in the `/etc/sudoers` file.   This is a good thread on the same - https://unix.stackexchange.com/questions/83191/how-to-make-sudo-preserve-path

Reply:
It seems like there is a little bit of confusion as to why `sudo -E PATH="$PATH"` isn't working and it might be due to the sudoers entry that we can't control in the github actions, so given that this approach seems to be working for Pradipta, we can go with it now and iterate on it later if necessary - thanks for the extra explanations!